docs/supported_versions.md

```diff
@@ -0,0 +1,206 @@
+# OCP Version Support
+
+## MCE hub (management or hosting) Cluster
+[Multi-Cluster Engine](https://docs.openshift.com/container-platform/4.17/architecture/mce-overview-ocp.html) (MCE), which is available through OpenShift's OperatorHub, require specific OCP versions for the Management Cluster to remain in a supported state.
+
+Each version documents its own support matrix. For example, 
+
+- [MCE 2.6](https://access.redhat.com/articles/7073030)
+- [MCE 2.5](https://access.redhat.com/articles/7056007)
+- [MCE 2.4](https://access.redhat.com/articles/7027079)
+
+As a heuristic, OCP versions that MCE supports are:
+
+- The latest, yet to be released version (N+1) of OpenShift
+- The latest GA version (N) of OpenShift
+- Two versions prior (N-2) to the latest GA version
+
+Versions of MCE can also be obtained with the Red Hat Advanced Cluster Management (ACM) offering. If you are running ACM, refer to product documentation to determine the bundled MCE version.
+
+## Hosted Clusters
+The OCP version of the management cluster does not affect the OCP version of a hosted cluster you can deploy. The hypershift operator that MCE installs creates a ConfigMap called `supported-versions` into  `hypershift` namespace, which describes the range of supported OCP versions that could be deployed. 
+
+Here is an example `supported-versions` ConfigMap:
+```
+apiVersion: v1
+data:
+    server-version: 2f6cfe21a0861dea3130f3bed0d3ae5553b8c28b
+    supported-versions: '{"versions":["4.17","4.16","4.15","4.14"]}'
+kind: ConfigMap
+metadata:
+    creationTimestamp: "2024-06-20T07:12:31Z"
+    labels:
+        hypershift.openshift.io/supported-versions: "true"
+    name: supported-versions
+    namespace: hypershift
+    resourceVersion: "927029"
+    uid: f6336f91-33d3-472d-b747-94abae725f70
+```
+
+It is important to note that you cannot install hosted clusters outside of this support version range. In the example above, hosted clusters using OCP release images greater than 4.17 cannot be created. If you want to deploy a higher version, you need to upgrade MCE to a new y-stream release to deploy a new version of the hypershift operator. Upgrading MCE to a new z-release updates the hypershift operator but does not upgrade its version.  
```

Comment:
> Upgrading MCE to a new z-release updates the hypershift operator but does not upgrade its version.

Can you add clarification to the last word, version? I could see folks getting confused on what exactly this means, i.e., is it the MCE version or the HyperShift Operator version or the OCP version.